server/a_socials/settings/logger_settings.py

```diff
@@ -0,0 +1,53 @@
+import dotenv
+import os
+
+os.makedirs('/tmp/log/andela_socials', mode=0o777, exist_ok=True)
```

Comment:
@kafuuma what's the purpose of this? I thought it's already been taken care of in `docker/server/Dockerfile` line 17